### DISCUSSION TITLE
Change the value of parameters in io.anserini.search.SearchArgs

### DISCUSSION THREAD

**Discussion by Pourbahman** [castorini/anserini#1697] [Q&A] [answered]
Hi,

I want to change the parameters of rm3 in my implementation. According to the following address
[https://github.com/castorini/anserini/blob/fbdd0087b33caaf3422ce2892a30fc3af5dbb468/src/main/java/io/anserini/search/SearchArgs.java#L22](url)
I can set the args.rm3 = true but I can not change the value of rm3_fbTerms using args.rm3_fbTerms[0] = '5'

Would you please guide me about it?

Thanks in advance,
Kind regards,
Zahra

**Comment by lintool** [comment, marked as answer]
Hi @Pourbahman - what exactly do you mean?

From the command line, you can configure RM3 parameters directly, e.g., https://github.com/castorini/anserini/blob/master/docs/regressions-disk45.md
